P2-Data-Model/content.md

```diff
@@ -75,7 +73,7 @@ The diagram above illustrates the different classes, their most important proper
 Let's discuss them briefly:
 
 - **User**
-A new *User* class is created whenever a user signs up. The *username* will store the user's display name: this name is used to log in and it displayed in the app when the user creates posts or likes other posts.
+A new *User* class is created whenever a user signs up. The *username* will store the user's display name: this name is used to login and is displayed in the app when the user creates posts or likes other posts.
```

Comment:
Pretty sure in this context it should be "to log in" because it describes an action.